README.md

```diff
@@ -201,8 +201,11 @@ entries()
 ```js
 // Provide an object which maps its properties and values
 // into the backing Map as keys and values.
+// You can also provide an array to the second argument
```

Comment:
s/an array to the second argument/an array as the second argument/ maybe? Up to you :-)